JBBCode/tests/ElementNodeTest.php

```diff
@@ -0,0 +1,80 @@
+<?php
+
+class ElementNodeTest extends PHPUnit_Framework_TestCase {
+    /** @var  JBBCode\ElementNode */
+    private $_elementNode;
+
+    protected function setUp()
+    {
+        $this->_elementNode = new JBBCode\ElementNode();
+    }
+
+    public function testConstructor()
+    {
+        $this->assertNull($this->_elementNode->getCodeDefinition());
+        $this->assertEmpty($this->_elementNode->getTagName());
+        $this->assertEmpty($this->_elementNode->getAttribute());
+        $this->assertEmpty($this->_elementNode->getChildren());
+        $this->assertEmpty($this->_elementNode->getAsText());
+        //$this->assertEmpty($this->_elementNode->getAsBBCode());
```

Comment:
Oh that, yes … must've slipped through. It appears an empty `ElementNode` is not rendering into an empty string as it expects to have a `tagName` set, so you're getting something like `[][/]` here. The proposed change in #56 would somewhat take care of that.